packages/account/src/Constants/routes-config.js

```diff
@@ -26,7 +26,7 @@ import {
 const Page404 = React.lazy(() => moduleLoader(() => import(/* webpackChunkName: "404" */ 'Modules/Page404')));
```

Comment:
can this file be in ts?

Reply:
Yes. We have a card for that section. Will make the change there